docs/en/latest/tutorials/the-hard-way.md

```diff
@@ -322,7 +322,7 @@ spec:
     spec:
       containers:
         - name: apisix
-          image: "apache/apisix:2.15-alpine"
+          image: "apache/apisix:2.15.0-alpine"
```

Comment:
I don't think any modifications are needed here. Since v2.15 is not yet EOL, it will continue to release new versions.

Reply:
@stubbornTanzhe any updates?

Reply:
> I don't think any modifications are needed here. Since v2.15 is not yet EOL, it will continue to release new versions.

BUT user will get error when he follow the doc,is that right?

Reply:
That means you got an error using "apache/apisix:2.15-alpine", but using "apache/apisix:2.15.0-alpine" installs fine, right?

This is not expected.

we should fix that

Reply:
> That means you got an error using "apache/apisix:2.15-alpine", but using "apache/apisix:2.15.0-alpine" installs fine, right?
> 
> This is not expected.
> 
> we should fix that

yes, that's what i mean~ if this pr is not suitable, please feel free to close it;)

Reply:
I will approve this PR first. Then create a new issue for tracking.

Reply:
https://github.com/apache/apisix-ingress-controller/issues/1671